discord_bot/discord_bot.py

```diff
@@ -144,24 +142,39 @@ def load_config(config_file_path):
                     self.logger.warning("Invalid Session")
                 elif message["op"] == Opcodes.DISPATCH:
                     event = message["t"]
```

Comment:
I actually might move all of this if-casing into a separate method called `dispatch_handler` or `event_handler` or something, whatever makes the most sense, to get `gateway_handler` to be smaller.